src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/BeneficialOwnersStatementType.java

```diff
@@ -12,12 +14,18 @@ public enum BeneficialOwnersStatementType {
         this.beneficialOwnersStatement = beneficialOwnersStatement;
     }
 
-    public static BeneficialOwnersStatementType findByBeneficialOwnersStatementTypeString(String beneficialOwnersStatement) {
-        for (BeneficialOwnersStatementType type: values()) {
-            if(type.beneficialOwnersStatement.equals(beneficialOwnersStatement)) {
-                return type;
-            }
-        }
-        return null;
-    }
+    /**
+     * Json creator allows non-matching enum values to be interpreted by the JSON serialiser as null
```

Comment:
Thanks for adding this. Comment not exactly correct though - the annotation simply says this method will be used when interpreting the values. Whether null is returned, or something else, is down to the method implementation.

Sorry to be pedantic, but I am like a human code compiler of plain English... 😆 .

Reply:
Updated the comment to refer to the method.